js/web/lib/onnxjs/backends/webgl/inference-handler.ts

```diff
@@ -134,16 +195,48 @@ export class WebGLInferenceHandler implements InferenceHandler {
     return this.createTextureDataFromTexture(layout, dataType, texture, tensor);
   }
 
-  /**
-   * Create a TextureData object, using the given texture.
-   * This function does not create new texture. Usually used in scenarios using texture sharing. (eg. Reshape)
-   * @param dataType the tensor data type
-   * @param texture the WebGLTexture object to share
-   * @param tensorId the tensor ID of the shared tensor data
-   */
-  createSharedTextureData(
-      layout: TextureLayout, dataType: Tensor.DataType, texture: WebGLTexture, tensorId?: Tensor.Id): TextureData {
-    return this.createTextureDataFromTexture(layout, dataType, texture, undefined, tensorId);
+  reshapeUnpacked(input: Tensor, reshapedDims: readonly number[]): Tensor {
```

Comment:
Should we move the two reshape functions to utils.ts?

Reply:
these 2 functions access texture cache directly so it's better to put inside inference handler class.